### PR TITLE
CONCF-520 | Send optimizely data to qualaroo

### PR DESCRIPTION
**Pull request by vforge**
Ping @bognix @rogatty @Warkot

### CODE REVIEW

File: server/views/_partials/qualaroo.hbs [hunk 1]
```diff
@@ -2,6 +2,18 @@
...
+
+		// user identity
+		// TODO when we'll support user login on mercury
+		// if (user_id) {
+		// 	_kiq.push(['identify', user_id]);
+		// }
+
+		// send some variables to qualaroo so we can target our audience
+		_kiq.push(['set', {
+			'optimizelyVariants': ((typeof optimizely === "object" && optimizely.variationMap) ? optimizely.variationMap : {})
```
Comment (bognix) [inline server/views/_partials/qualaroo.hbs]: move second arg that you pass to qualaroo to separate object. It would increase readability.

Comment (rogatty) [inline server/views/_partials/qualaroo.hbs]: Can you shorten this line? For example by using a variable.

**Comment by bognix** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by vforge** [comment]
Pinging @bognix @rogatty again - I've updated code a bit.

File: server/views/_partials/qualaroo.hbs [hunk 2]
```diff
@@ -1,7 +1,30 @@
 {{#if qualarooScript}}
 	<script>
 		// qualaroo configuration
-		var _kiq = _kiq || [];
+		var _kiq = [],
+			qualarooConfig = {};
+
+		(function(){
+			setTimeout(function(){
```
Comment (rogatty) [inline server/views/_partials/qualaroo.hbs]: Can you add a comment explaining why do we need setTimeout?

Comment (vforge) [inline server/views/_partials/qualaroo.hbs]: Yes :|

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :hamster:

**Pull request merged**